Content.Shared/Armor/ArmorDamageSystem.cs

```diff
@@ -0,0 +1,21 @@
+using Content.Shared.Damage;
+using Content.Shared.Inventory;
+using Content.Shared.Armor;
+
+namespace Content.Shared.ArmorDamage
+{
+    public sealed class ArmorSystem : EntitySystem
+    {
+
+        public override void Initialize()
+        {
+            base.Initialize();
+
+            SubscribeLocalEvent<ArmorComponent, InventoryRelayedEvent<DamageModifyEvent>>(OnDamageModify);
+        }
+        private void OnDamageModify(EntityUid uid, ArmorComponent component, InventoryRelayedEvent<DamageModifyEvent> args)
+        {
+            args.Args.Damage = DamageSpecifier.ApplyModifierSet(args.Args.Damage, component.Modifiers);
+        }
+    }
+}
```

Comment:
1. Please use file-scoped namespaces
2. The server system should be inheriting this
3. You should move the examine here
4. ArmorComponent isn't actually networked